internal/core/util_test.go

```diff
@@ -61,31 +59,18 @@ type DrandTestScenario struct {
 	resharedNodes []*MockNode
 }
 
-// BatchNewDrand returns n drands, using TLS or not, with the given
+// BatchNewDrand returns n drand daemons, with the given
 // options. It returns the list of Drand structures, the group created,
-// the folder where db, certificates, etc are stored. It is the folder
+// the folder where db, etc are stored. It is the folder
 // to delete at the end of the test. As well, it returns a public grpc
 // client that can reach any drand node.
-//
-//nolint:funlen // This is a test function
-func BatchNewDrand(
-	t *testing.T,
-	currentNodeCount,
-	n int,
-	insecure bool,
-	sch *crypto.Scheme,
-	beaconID string,
-	opts ...ConfigOption,
-) (
-	daemons []*DrandDaemon, drands []*BeaconProcess, group *key.Group, dir string, certPaths []string,
-) {
+func BatchNewDrand(t *testing.T, currentNodeCount, n int,
+	sch *crypto.Scheme, beaconID string,
+	opts ...ConfigOption) (daemons []*DrandDaemon, drands []*BeaconProcess, group *key.Group, dir string) {
 	t.Logf("Creating %d nodes for beaconID %s\n", n, beaconID)
 	var privs []*key.Pair
```

Comment:
can remove this predeclaration and just use `:=` now I think

Reply:
wouldn't want to override my lovely named returns, would I?